src/doc/trpl/README.md

```diff
@@ -0,0 +1,35 @@
+% The Rust Programming Language
+
+Welcome! This book will teach you about [the Rust Programming
+Language](http://www.rust-lang.org/). Rust is a modern systems programming
+language focusing on safety and speed. It accomplishes these goals by being
+memory safe without using garbage collection.
+
+"The Rust Programming Language" is split into three sections, which you can
+navigate through the menu on the left.
+
+## Basics
+
+This section is a linear introduction to the basic syntax and semantics of
+Rust. It has individual sections on each part of Rust's syntax, and cumulates
+in a small project: a guessing game.
+
+After reading "Basics," you will have a good foundation to learn more about
+Rust, and can write very simple programs.
+
+## Intermediate
+
+This section contains individual chapters, which are self-contained. They focus
+on specific topics, and can be read in any order.
+
+After reading "Intermediate," you will have a solid understanding of Rust,
+and will be able to understand most Rust code and write more complex programs.
+
+## Advanced
+
+In a similar fashion to "Intermediate," this setion is full of individual,
```

Comment:
This PR is just a bulk move of all the docs; the actual contents aren't under review here (feel free to submit a seperate issue or PR).